doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -467,58 +467,309 @@ The following table indicates the software maturity levels of the support for ea
+        - --
+      * - **Periodic Advertising Sync Transfer**
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
         - Supported
         - Experimental
+        - Supported
+        - --
+        - --
+        - --
+        - --
+      * - **Isochronous Channels**
+        - Supported\ :sup:`2`
+        - Supported\ :sup:`2`
+        - Supported
+        - Supported\ :sup:`2`
+        - Supported
+        - Supported\ :sup:`2`
+        - Supported
         - Experimental
+        - Supported
+        - --
+        - --
+        - --
+        - --
+      * - **Direction Finding**\ :sup:`3`
+        - --
+        - --
+        - --
+        - --
+        - Supported
+        - --
+        - Supported
+        - --
         - --
         - --
         - --
         - --
-      * - **LE Coded PHY**
         - --
+      * - **LE Power Control**
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
         - Supported
         - Experimental
+        - Supported
         - --
+        - --
+        - --
+        - --
+      * - **Connection Subrating**
+        - Supported
+        - Supported
+        - Supported
+        - Supported
         - Supported
         - Supported
         - Supported
         - Experimental
+        - Supported
+        - --
+        - --
+        - --
+        - --
+      * - **Channel Sounding**
+        - --
+        - --
+        - --
+        - --
+        - --
+        - --
+        - --
+        - --
         - Experimental
         - --
         - --
         - --
         - --
-      * - **LLPM**
+      * - **GATT Database Hash**
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Experimental
+        - Supported
         - --
         - --
         - --
+        - --
+      * - **Enhanced ATT**
+        - Supported
+        - Supported
+        - Supported
         - Supported
         - Supported
         - Supported
+        - Supported
+        - Experimental
+        - Supported
         - --
+        - --
+        - --
+        - --
+      * - **L2CAP Connection Oriented Channels**
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Experimental
+        - Supported
+        - --
+        - --
+        - --
+        - --
+
+| [1]: Concurrent central, observer, peripheral, and broadcaster roles with up to 20 concurrent connections along with one Observer and one Broadcaster (subject to RAM availability)
+| [2]: Do not support encrypting and decrypting the Isochronous Channels packets
+| [3]: Only AoA transmitter is supported
+
+The following table indicates the software maturity levels of the support for each proprietary Bluetooth feature:
```

Comment:
I will keep a separate table for the proprietary features before we know how to separate/group them in the same table